### ISSUE TITLE
Illustration Artwork from Ricardo

### ISSUE THREAD

**Issue by bencallahan** [mattgraham/frontendthanks#6]
This is the Illustrator file Ricardo provided of the artwork he could find. He said he’s missing 2010 speakers. I’ve also asked him to provide the illustration of Dan.

https://www.dropbox.com/s/gedhtof1b4pq6tb/frontend.zip

**Comment by mattgraham** [comment]
Awesome; i still have mine around here somewhere also. Feel free to toss out concepts and ideas on this.

**Comment by mattgraham** [comment]
or maybe I dont have mine. Looks like we're missing 2013 also?

**Comment by mattgraham** [comment]
(and cherrie would be nice)

**Comment by bencallahan** [comment]
Here’s Dan:

https://s3.amazonaws.com/mixture-mixed/161/355/_site/public/images/front-end-dan.svg

**Comment by chriscoyier** [comment]
Maybe we can get Ricardo to put every single character (just heads?) he's Illustrated for FED into one giant graphic and that's what we print on the shirt? Unless it's too busy and ridiculous looking. We could use that for the site somewhere too. 

Do we allow people to opt-out?

**Comment by chriscoyier** [comment]
Fabio Sasso - 2009
Chris Coyier - 2009, 2011, 2013, 2014 (PDX)
Grant Friedman - 2009
Andrew Maier - 2009
Jonathan Longnecker - 2009
John Ashenden - 2009
Kevin Hale - 2009
Larissa Meek  - 2010
Lea Alcantara - 2010
Niki Brown - 2010
Meagan Fisher - 2010
Jina Bolton - 2010
Sarah Parmenter - 2010, 2012
Adelle Charles - 2010
James May - 2011
Stephanie Fields - 2011
Ryan Singer - 2011
Doug Neiner - 2011, 2013
Ethan Marcotte - 2011
Brandon Jones - 2011
Allison House - 2011
Jason Vanlue - 2012, 2014 (PDX)
Darcey Clarke - 2012
Giovanni DiFeterici - 2012
Bermon Painter - 2012, 2014 (PDX)
Carl Smith - 2012, 2014 (St. Pete)
Dave Rupert - 2012, 2013
Nick Walsh - 2013
Drew Barontini - 2013, 2014 (St. Pete)
Matt Graham - 2013
Michael Parenteau - 2013
Jenn Lukas - 2013
Ben Callahan - 2013, 2014 (PDX)
Cameron Moll - 2013
Daniel Ryan - 2014 (St. Pete), 2014 (PDX)
Rachel Smith - 2014 (St. Pete), 2014 (PDX)
Matthew Carver - 2014 (St. Pete), 2014 (PDX)
Erica Walker - 2014 (St. Pete)
Amanda Cheung - 2014 (St. Pete)
Andi Graham - 2014 (St. Pete)
Robb Harr - 2014 (St. Pete)
Travis Miller - 2014 (St. Pete)
Andrew Norcross - 2014 (St. Pete)
Noah Stokes - 2014 (St. Pete)
Mina Markham - 2014 (St. Pete)
Jason Beaird - 2014 (St. Pete)

... doesn't include next-day talks ...

**Comment by mattgraham** [comment]
Love the idea! Im sure we can find the images online also.

**Comment by mattgraham** [comment]
maybe we just put frontend logo etc. on the front and a list of names etc on the back as if it was a class graduation shirt etc.

**Comment by bencallahan** [comment]
This is great, guys. Unfortunately, Ricardo can’t seem to find all the speakers. Seems like we should do something other than all the faces if we can’t actually get ALL the faces...

Jeremy Loyd, from my team, is working on a design. Will share this with him as well. We have to get the shirt stuff figured out immediately, as the CB guys are heading out of the country next week!

**Comment by chriscoyier** [comment]
I could dig up emails for everyone on that list probably. Should we send one to the whole list asking folks if they would like to write a short "thank you" that we ultimately use on the site?

**Comment by bencallahan** [comment]
Would be great to get them to submit a thank you or a story, etc. That would fit nicely into the site.

Could also ask them to share the shirt URL once it’s live?

**Comment by chriscoyier** [comment]
If the plan is to allow people to WRITE thank yous that go on the site, maybe we'll wait on that mass-email and have them use that system? Or should I send it now and ask them to just shoot over the thank-you as text? 

Maybe the form on the site has some simple form on it that saves the stuff to https://www.firebase.com/ ? Or keep it wicked simple and use a Disqus thread ?

**Comment by mattgraham** [comment]
I say lets launch with a good number of them; if you want to request them that works; im hoping we can get this live early this week. It's getting closer.

**Comment by chriscoyier** [comment]
Not able to dig up emails for:

http://larissameek.com/say-hello/
https://twitter.com/lilsooner
http://drewbarontini.com/
BEN http://bencallahan.com/
https://dryan.com/contact/
https://twitter.com/ebwalker101
https://twitter.com/robertHarr
http://monooptical.com/
https://twitter.com/makedesign

But I got everybody else. Should I just send like:

> Hi,
> 
> Forgive the intrusion here, we're trying to do something nice for a nice fella you know: Dan Denney. Coming up later this week is the last-ever Front End Design Conference. You've spoken at one before. If you have a good memory of that, or some nice memory to share, can you EMAIL IT? (OR SOMETHING?). We're going to make a little thank-you site for Dan (and gang) for doing a good thing over the years.
> 
> -Chris

?

**Comment by bencallahan** [comment]
Chris,

Yeah, that's good. Maybe also stress the urgency of the deadline?

Ben Callahan: ben@heysparkbox.com
Rob Harr: rob@heysparkbox.com

b

On Monday, August 18, 2014, Chris Coyier notifications@github.com wrote:

> Not able to dig up emails for:
> 
> http://larissameek.com/say-hello/
> https://twitter.com/lilsooner
> http://drewbarontini.com/
> BEN http://bencallahan.com/
> https://dryan.com/contact/
> https://twitter.com/ebwalker101
> https://twitter.com/robertHarr
> http://monooptical.com/
> https://twitter.com/makedesign
> 
> But I got everybody else. Should I just send like:
> 
> Hi,
> 
> Forgive the intrusion here, we're trying to do something nice for a nice
> fella you know: Dan Denney. Coming up later this week is the last-ever
> Front End Design Conference. You've spoken at one before. If you have a
> good memory of that, or some nice memory to share, can you EMAIL IT? (OR
> SOMETHING?). We're going to make a little thank-you site for Dan (and gang)
> for doing a good thing over the years.
> 
> -Chris
> 
> ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mattgraham/frontendthanks/issues/6#issuecomment-52586783
> .

## 

_Ben Callahan__Sparkbox, President_
123 Webster Street, Studio 2
Dayton, OH 45402
937.401.0915
www.seesparkbox.com

**Comment by chriscoyier** [comment]
sent

**Comment by mattgraham** [comment]
I've added a speakers page; but think we can do much more on it with the illustrations. Maybe we'll open a new issue for it as we launch.
